mysql-test/suite/rpl/t/bug84415.test

```diff
@@ -0,0 +1,265 @@
+-- source include/master-slave.inc
```

Comment:
Sleeps in test are always a red flag, but here they are almost fine: it's OK to use sleep to ensure that at least this much time has passed, i.e. to create some slave delay. But it's not OK to 1) have upper bound for time passed, it's arbitrarily large on arbitrarily slow machine, i.e. $upper_bound must go away; 2) use sleep for synchronisation: "--real_sleep 1 # give mts checkpoint chance to update slave status" must be replaced by some condition wait or some other sync